README.md

```diff
@@ -1,16 +1,74 @@
-# Title
+# Writing Dynamic Programs
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
++ Write a case statement
++ Recognize the comment character
++ Compare running a Ruby file versus using IRB
++ Run a Ruby program in the Learn In-Browser IDE
++ Write an if...end statement
++ Write an if...else...end Statement
++ Write an if...elsif...else...end Statement
++ Use puts statement to send content to the screen
```

Comment:
This probably isn't as major when it comes to video content, but I try to catch myself when I write learning goals that begin with "Use ...." Instead, I try to find what action the thing is being used for and bring that to the front. For example, here I would rewrite this as: "Send content to the screen with puts statement." It emphasizes the action rather than tool.

Reply:
Thanks, Jen. I actually like the "Use" word here, as it helps to map to a level of Bloom's taxonomy. In this case, we're trying to get to the "Application" stage - can the user apply what they've learned? https://www.teachthought.com/learning/what-is-blooms-taxonomy-a-definition-for-teachers/

So, after this video, my hope is students will be able to use a puts statement to send content to the screen in their terminal. Having said all this, definitely open to different approaches if you feel strongly one way or another.